src/Fieldtypes/HasManyFieldtype.php

```diff
@@ -29,6 +29,11 @@ protected function configFieldItems(): array
     // Pre-process the data before it gets sent to the publish page
     public function preProcess($data)
     {
+        $resourceHandle = request()->route('resourceHandle');
```

Comment:
I've not tested this yet but I'm wondering if this is going to work in all cases. 

For example: it might work for entries/terms but not sure if this check will work if it's a HasMany on another model? Eg. Post HasMany Tags..

Reply:
Not sure I follow what you mean - `resourceHandle` should be available on any Runway defined resources/models - it was throwing an error when it wasn't defined (eg on entries/terms). Maybe I'm missing something...

Reply:
Ignore me - it's probably me thinking up problems. I'll give it a proper test in the next few days. 

I appreciate the PR(s)!